src/components/header/style.module.scss

```diff
@@ -17,12 +18,17 @@
     display: flex;
     align-items: center;
     justify-content: space-between;
+	color: $color;
 
     @include respond-to('desktop') {
       width: 50%;
       margin: 0 auto;
     }
-
+	
+	.textcolor {
```

Comment:
You defining this .textcolor but then on the component there's nothing using this class. Its just easier to define the color on the container, and put color: inherit on the css

Reply:
Will change!